### PR TITLE
[docs] Update docker guide for Selenium 4

### LINKED ISSUE #3049: Update selenium version in CI integration docs
Vividus has moved onto selenium 4, but we still see mention of selenium 3 in docs https://docs.vividus.dev/vividus/latest/user-guides/pipeline-with-proxy.html so it needs to be updated

### PR DESCRIPTION
**Pull request by uarlouski**
Closes #3049

### CODE REVIEW

File: docs/modules/user-guides/pages/pipeline-with-proxy.adoc [hunk 1]
```diff
@@ -53,7 +56,7 @@ pipeline {
     stage('Run Tests Using Docker') {
         steps {
                 sh "docker ps -a"
-                withDockerContainer(image: 'openjdk:16.0.2-oraclelinux8', args: "--net ${vividusNetwork} --name ${vividusProxyHost}") {<1> <4>
+                withDockerContainer(image: 'adoptopenjdk/openjdk16', args: "--net ${vividusNetwork} --name ${vividusProxyHost}") {<1> <4>
```
Comment (ikalinin1) [inline docs/modules/user-guides/pages/pipeline-with-proxy.adoc:59]: 17 ?

File: docs/modules/user-guides/pages/pipeline-with-proxy.adoc [hunk 2]
```diff
@@ -5,17 +5,20 @@ def vividusNetwork = 'vividus-grid'<1>
...
+def seleniumVersion = '4.4.0'<5>
 def overridings = """
                         proxy.host=${vividusProxyHost}<4>
                         proxy.connectable-host=${vividusProxyHost}<4>
                         selenium.grid.enabled=true
                         selenium.grid.url=http://${vividusHub}:4444/wd/hub<2>
                         selenium.grid.capabilities.platformName=LINUX
+                        selenium.grid.capabilities.browserName=chrome
+                        configuration.suites=web_app
+                        configuration.profiles=web/headless/chrome
```
Comment (ikalinin1) [inline docs/modules/user-guides/pages/pipeline-with-proxy.adoc:17]: Refactor to use web/docker

File: docs/modules/user-guides/pages/pipeline-with-proxy.adoc [hunk 3]
```diff
@@ -5,17 +5,18 @@ def vividusNetwork = 'vividus-grid'<1>
 def vividusHub = 'vividus-hub'<2>
 def vividusBrowser = 'vividus-browser'<3>
 def vividusProxyHost = 'vividus-proxy-host'<4>
-def seleniumVersion = '3.141.59-20210804'<5>
+def seleniumVersion = '4.4.0'<5>
```
Comment (valfirst) [inline docs/modules/user-guides/pages/pipeline-with-proxy.adoc:8]: `4.5.0` ?

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/user-guides/pages/pipeline-with-proxy.adoc [hunk 4]
```diff
@@ -5,17 +5,18 @@ def vividusNetwork = 'vividus-grid'<1>
...
 def vividusBrowser = 'vividus-browser'<3>
 def vividusProxyHost = 'vividus-proxy-host'<4>
-def seleniumVersion = '3.141.59-20210804'<5>
+def seleniumVersion = '4.5.0'<5>
 def overridings = """
                         proxy.host=${vividusProxyHost}<4>
                         proxy.connectable-host=${vividusProxyHost}<4>
-                        selenium.grid.enabled=true
+                        configuration.suites=web_app
+                        configuration.profiles=web/docker
```
Comment (ikalinin1) [inline docs/modules/user-guides/pages/pipeline-with-proxy.adoc:13]:
```suggestion
                        configuration.profiles=web/docker,web/desktop/chrome

```

**Review by valfirst — APPROVED** [review]

**Pull request merged**